newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -319,6 +320,21 @@ export const GameDetailsDialog = ({
                 multiline
                 rows={5}
               />
+              <SelectField
+                disabled
+                fullWidth
+                floatingLabelText={
+                  <Trans>Device orientation (for iOS and Android)</Trans>
```

Comment:
This can be renamed to show this is not only ios/android. Maybe `Device orientation (for mobile)`

Reply:
I would also add "for Liluo.io" maybe too? What do you think?

Reply:
The project properties view rework will probably make it clear. As every field is actually for Liluo.io.

Reply:
Ok!